GitTfs/Commands/InitBranch.cs

```diff
@@ -135,28 +137,48 @@ private IGitTfsRemote InitBranchSupportingRename(string tfsBranchPath, string gi
 
             _stdout.WriteLine("Branches to Initialize successively :");
             foreach (var branch in creationBranchData)
-                _stdout.WriteLine("-" + branch.TfsBranchPath + " (" + branch.RootChangeset + ")");
+            {
+                var outputMessage = "-" + branch.TfsBranchPath + " (Source C" + branch.SourceBranchChangesetId;
+
+                if (branch.TargetBranchChangesetId > -1)
+                    outputMessage += " -> Target C" + branch.TargetBranchChangesetId;
+
+                outputMessage += ")";
+
+                _stdout.WriteLine("-" + outputMessage);
+            }
 
-            IGitTfsRemote tfsRemote = null;
+            IGitTfsRemote branchTfsRemote = null;
```

Comment:
For clarity.
The whole purpose of this changeset is to better support detection of the actual branch point of a given child branch and its source changeset in the parent branch.
The IGitTfsRemote we work with isn't just a random "remote" or even a "remote" referring to the repository base - it's specific to the branch being init'd.